eiffel-vocabulary/EiffelIssueVerifiedEvent.md

```diff
@@ -35,6 +27,24 @@ __Legal targets:__ [EiffelIssueDefinedEvent](../eiffel-vocabulary/EiffelIssueDef
 __Multiple allowed:__ Yes  
 __Description:__ Identifies an issue that this event pertains to.
 
+### SUCCESFUL_ISSUE
+__Required:__ Yes  
+__Legal targets:__ [EiffelIssueDefinedEvent](../eiffel-vocabulary/EiffelIssueDefinedEvent.md)  
+__Multiple allowed:__ Yes  
+__Description:__ Identifies an issue that this event pertains to.
+
+### FAILED_ISSUE
+__Required:__ Yes  
```

Comment:
We should only require that one of `SUCCESSFUL_ISSUE`, `FAILED_ISSUE`, `INCONCLUSIVE_ISSUE` be required. So for each of these, mark them as `__Required:__ No` and make a note in the description at the top of the file.